source/ed/ed_persistor.cpp

```diff
@@ -726,6 +730,35 @@ void EdPersistor::insert_lines(const std::vector<types::Line*>& lines){
 
 }
 
+void EdPersistor::insert_line_groups(const std::vector<types::LineGroup*>& groups, const std::vector<types::LineGroupLink>& group_links){
+    this->lotus.prepare_bulk_insert("navitia.line_group", {"id","uri","name"});
+    for(const auto& line_group: groups) {
+        this->lotus.insert({
+            std::to_string(line_group->idx),
+            navitia::encode_uri(line_group->uri),
+            line_group->name
+        });
+    }
+    this->lotus.finish_bulk_insert();
+
+    this->lotus.prepare_bulk_insert("navitia.line_group_link", {"group_id","line_id","is_main_line"});
```

Comment:
Why is there a `is_main_line` in this relation table, and not an id in the `line_group` table with the id of the main line, as it is in the NTFS? It seems cleaner and simpler to me. (Same for `types::LineGroupLinks`)